packages/esm-billing-app/src/m-pesa/mpesa-resource.tsx

```diff
@@ -1,39 +1,81 @@
-import { Buffer } from 'buffer';
+export type RequestStatus = 'INITIATED' | 'COMPLETE' | 'FAILED' | 'NOT-FOUND';
```

Comment:
```suggestion
export type RequestStatus = 'INITIATED' | 'COMPLETE' | 'FAILED' | 'NOT-FOUND';
```
Ideally this should be in the types file https://github.com/palladiumkenya/kenyaemr-esm-3.x/blob/main/packages/esm-billing-app/src/types/index.ts

Reply:
On it